scripts/core/editor3/components/suggestions/SuggestionPopup.jsx

```diff
@@ -33,9 +38,20 @@ export class SuggestionPopup extends Component {
                 this.setState({error: gettext('An error occured, please try again.')});
             });
     }
+
+    onAccept() {
+        // TODO: close widget
```

Comment:
not sure if you have to close it, when you accept the entity should be gone and thus the popup should be gone, same on reject..

Reply:
Yes, somehow it is not gone.

Reply:
FYI this line is responsible for closing it https://github.com/superdesk/superdesk-client-core/blob/master/scripts/core/editor3/components/HighlightsPopup.jsx#L271

Reply:
After I rebased it is working fine.